browser/services/electron.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
```

Comment:
Obviously, there is bettera way to implement this. Instead of creating new mediator between electron and application, controllers should get Electron instance through AngularJS DI. Then in unit tests it would be much easier to achieve higher coverage by injecting fake electron implementation using Sinon.JS mocks/stubs.